### PR TITLE
add governance pages

### PR DESCRIPTION
**Pull request by MrPowers**

### CODE REVIEW

File: docs/governance/mrpowers-io-org.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# mrpowers-io GitHub organization
+
+The mrpowers-io GitHub organization is owned by [Matthew Powers](https://www.linkedin.com/in/matthew-powers-cfa/) and [Sem Sinchenko](https://www.linkedin.com/in/semyon-a-sinchenko/) ("org owners").
```
Comment (SemyonSinchenko) [inline docs/governance/mrpowers-io-org.md:3]: I would like to have here a link to my github account, not Linkedin. The reason is that not all of potential contributors may have linkedin account but all of the should have a git account.

File: docs/governance/mrpowers-io-org.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# mrpowers-io GitHub organization
+
```
Comment (SemyonSinchenko) [inline docs/governance/mrpowers-io-org.md:2]: What do you think about putting contact email accounts here?

File: docs/governance/mrpowers-io-org.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+The org owners are responsible for overseeing projects and reviewing project reports on a quarterly basis.
+
+## Overseeing projects
+
+The org owners are responsible for voting on new projects that will be added in the mrpowers-io organization.
+
+The org owners are also responsible for monitoring the projects at a high-level including code quality, documentaion, the README, and repo activity.
+
+mrpowers-io is generally for repos that are being actively developed and used.
```
Comment (SemyonSinchenko) [inline docs/governance/mrpowers-io-org.md:13]: The same text is in line 35 too...

File: docs/governance/mrpowers-io-org.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+This ensures the package can continue being published in the event of an extreme event like a hardware failure, losing keys, or getting hacked.
+
+## Quarterly project reports
+
+Repo maintainers should submit project reports on a quarterly basis showing key metrics like the number or new contributors, number of new stars, PyPI downloads, etc.
+
+The report should explain how the repo maintainers plan to grow the project in the upcoming quarter.
+
+The mrpowers-io org is generally for projects that are being actively maintained with maintainers that are ambitiously growing the user base.
+
```
Comment (SemyonSinchenko) [inline docs/governance/mrpowers-io-org.md:36]: I think that we should add a section about projects retirement. For example, if one day `chispa` will be moved to core pyspark we should archive it.

File: docs/governance/mrpowers-io-org.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+## Managing repo permissions
+
+The org owners will have to manage repo permissions from time-to-time.
+
+The org owners may need to add a new maintainer to a repo if other repo owners don't have permissions, for example.
+
+## Overseeing package manager permissions
+
+Repos in the mrpowers-io organization will often be bundled and distributed via a package manager like Maven or PyPI.
```
Comment (SemyonSinchenko) [inline docs/governance/mrpowers-io-org.md:23]:
```suggestion
Repos in the mrpowers-io organization will often be bundled and distributed via a package manager like Maven, PyPI or Cargo.
```

**Review by SemyonSinchenko — COMMENTED** [review]
Review of org

File: docs/governance/mrpowers-io-org.md (hunk 2 above)
Comment (MrPowers) [reply docs/governance/mrpowers-io-org.md:2]: Think it's best if they pull these from our GitHub (so they're not on the internet for spammers).

File: docs/governance/mrpowers-io-org.md (hunk 1 above)
Comment (MrPowers) [reply docs/governance/mrpowers-io-org.md]: Updated.

File: docs/governance/mrpowers-io-org.md (hunk 3 above)
Comment (MrPowers) [reply docs/governance/mrpowers-io-org.md]: Removed.

File: docs/governance/mrpowers-io-org.md (hunk 4 above)
Comment (MrPowers) [reply docs/governance/mrpowers-io-org.md:34]: Yea, this is outlined on the mrpowers-io-repos page.

File: docs/governance/mrpowers-io-org.md (hunk 5 above)
Comment (MrPowers) [reply docs/governance/mrpowers-io-org.md]: Updated this.

**Comment by MrPowers** [comment]
@SemyonSinchenko - thanks for reviewing.  I addressed your comments.

**Pull request merged**